src/android/com/perfectco/cordova/TlsPskClientSocket.java

```diff
@@ -21,6 +22,7 @@ public void connect(String host, int port) throws IOException {
     }
 
     Socket socket = new Socket(host, port);
+    socket.setSoTimeout(SOCKET_TIMEOUT_MS);
```

Comment:
So I was imagining we'd use something like [TCP_USER_TIMEOUT](https://man7.org/linux/man-pages/man7/tcp.7.html) to cause a timeout on send if the message wasn't acked within ~5sec.
However, it appears that socket option is linux-only at the moment, and there's no Java API to set it.

This sets a timeout on receive instead, and should achieve the same end, but requires that the other end faithfully send pings faster than 5 sec.

Looking at the java docs, I just found [SO_KEEPALIVE](https://docs.oracle.com/javase/8/docs/api/java/net/SocketOptions.html#SO_KEEPALIVE) and [family](https://docs.oracle.com/en/java/javase/16/docs/api/jdk.net/jdk/net/ExtendedSocketOptions.html#TCP_KEEPCOUNT)